state/execution.go

```diff
@@ -326,11 +326,12 @@ func updateValidators(currentSet *types.ValidatorSet, abciUpdates []abci.Validat
 	if err != nil {
 		return err
 	}
-
+	// store the current total voting power before applying any updates
+	totalVotingPower := currentSet.TotalVotingPower()
 	// these are tendermint types now
 	for _, valUpdate := range updates {
 		if valUpdate.VotingPower < 0 {
-			return fmt.Errorf("Voting power can't be negative %v", valUpdate)
+			return fmt.Errorf("voting power can't be negative %v", valUpdate)
```

Comment:
please revert. we use Capitalized errors, I think :)

Reply:
That's weird because then they don't read well if you wrap them: https://github.com/golang/go/wiki/CodeReviewComments#error-strings
But sure will revert.